frontend/src/Components/ContactForm/ContactForm.jsx

```diff
@@ -1,27 +1,34 @@
-import React from "react";
+import React, { useState } from "react";
 import { Button, TextField } from "@material-ui/core";
 import { useTranslation } from "react-i18next";
 import { makeStyles } from "@material-ui/core/styles";
 import { string as yupstring, object as yupobject } from "yup";
 import { useForm } from "react-hook-form";
 import { sendEmail } from "Utils/Requests";
+import { CustomSnackbar } from "Components";
 
 import "./ContactForm.scss";
 
-const useInputStyles = makeStyles(theme => ({
+const useInputStyles = makeStyles((theme) => ({
```

Comment:
We should probably have this in SCSS, mixing up styling method's will add levels of confusion.

Reply:
They're in here because the styles are conditional... I suppose a class could be added to them instead, but this was my first idea

Reply:
Do you have an alternative?

Reply:
Yeah, I've been using conditional classes to add CSS up until now. It would still use all your ternary statements so I think changes would be minimal.